truss/util/path.py

```diff
@@ -153,12 +153,15 @@ def is_ignored(
         for pattern in patterns:
             if original_path.is_dir() and pattern.endswith("/"):
                 pattern = pattern.rstrip("/")
-                if fnmatch.fnmatch(path.name, pattern):
+                if fnmatch.fnmatch(str(path), pattern) or fnmatch.fnmatch(
```

Comment:
I think a small comment here with examples of things that would match would help readers a lot, it's not obvious what fnmatch is doing here.

Also per @bolasim's comments, I think it would be good to add to the `test_is_ignored` test (https://github.com/basetenlabs/truss/blob/main/truss/tests/util/test_path.py#L53), since I don't think this behavior is tested for there.